incubator/riemann/templates/configmap.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-configmap
```

Comment:
Should be: `name: {{ template "riemann.fullname" . }}`

Reply:
Add standard labels.

Reply:
I've renamed the configMap to use the template and drop the -configmap suffix.
I've added the missing standard labels to the configMap (and the pod defined in the deployment)